src/models/unet.py

```diff
@@ -39,7 +40,7 @@ def __init__(
         self.layers = nn.ModuleList(layers)
 
         # final layer to match the depth map dimensions
-        self.conv_reshape = nn.Conv2d(in_channels=1, out_channels=1, kernel_size=(3,11), stride=1, padding=1, dilation=8)
+        self.conv_reshape = nn.Conv2d(in_channels=1, out_channels=1, kernel_size=(3,11), stride=(1, num_stack_horizontal), padding=(1, num_stack_horizontal), dilation=(8, 8*num_stack_horizontal))
```

Comment:
@jeswan did you check that if you add this line the normal code still runs with and without stack horizontal? will try run the 2 experiments you tried this on as well as running same code with stacking set to False

Reply:
Yes! I checked it works with the default arguments

Reply:
thats awesome!